contracts/core/UserOperationLib.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
please don't refactor UseROperation as part of "bls aggregator fix", but in a separate PR.